ShoeBox/Classes/GUI/LocationScreen/ViewController/LocationDetailViewController.swift

```diff
@@ -27,7 +27,13 @@ class LocationDetailViewController: UITableViewController, MKMapViewDelegate {
        
         centerMapOnLocation()
         
-        directions = NSLocalizedString("shoebox_distance", comment: "") + ": " + NSLocalizedString("shoebox_calculating", comment: "")
+        if (LocationManager.sharedInstance.currentLocation != nil) {
```

Comment:
Please use "if let currentLocation = LocationManager.sharedInstance.currentLocation" instead of 
if (LocationManager.sharedInstance.currentLocation != nil)